games-emulation/jgemu/jgemu-1.0.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Meta ebuild for the Jolly Good API"
+HOMEPAGE="https://jgemu.gitlab.io/"
+
+LICENSE="metapackage"
+SLOT="1"
+KEYWORDS="~amd64"
+IUSE="+bsnes +cega +gambatte +jollycv +mednafen melonds +mgba +nestopia +prosystem sameboy +vecx"
+
+DEPEND="
```

Comment:
I think RDEPEND (pure RDEPEND) for this (I can fix this on merge)

Reply:
Thanks for catching this, I must have copied other metapackages like `games-emulation/mupen64plus` incorrectly...